misc/rtc.md

```diff
@@ -28,29 +28,31 @@ With the latter, we're able to block very specific regions of our choice that ma
 
 For the rest of this guide, we're going to assume you've tested option 1(`DisableRtcChecksum`) and it didn't work or you're having issues with EfiBoot also writing to RTC. To get started, we should first introduce a few ideas:
 
-* RTC will generally have regions spanning from 0 to 255
+* RTC will have regions spanning from 0 to 255
 * These regions will be in the hexadecimal counting system so in reality will be 0x00-0xFF
 * To omit bad regions, we use the boot-arg `rtcfx_exclude=00-FF`
-  * Replace `00-FF` with your bad region
-  * Reminder boot-args is located under NVRAM -> Add in your config.plist
+  * Replace `00-FF` with your bad region (or regions)
+  * Reminder that `boot-args` is located under `NVRAM -> Add -> 7C436110-AB2A-4BBB-A880-FE41995C9F82` in your config.plist
   * This will also require you to have [RTCMemoryFixup](https://github.com/acidanthera/RTCMemoryFixup/releases/tag/1.0.6) in your config.plist and EFI/OC/Kexts folder.
 * There can be multiple bad regions
 * To find the bad region, we'll want to split out search into chunks
 
 Regarding splitting out chunks, what we'll be doing is omitting chunks of RTC regions until we've narrowed down far enough to the exact spot that's bad. You can see the below on how to start:
 
 1. Testing RtcMemoryFixup
-  * To start, we'll want to use `rtcfx_exclude=00-FF` in boot-args. This will tell us whether your CMOS errors are RTC related
+  * To start, you'll need to add `rtcfx_exclude=00-FF` in boot-args. If after a reboot the RTC errors seems solved, this will tell you whether your CMOS errors are RTC related
 2. Split 0x00-0xFF into 2
   * 0x00-0x7F and 0x80-0xFF
+      * write down the excluded range which fixes the RTC errors and proceed by splitting more into chunks
+      * e.g. `rtcfx_exclude=00-7F`fixes the RTC errors so you're gonna split it by half and don't consider more `rtcfx_exclude=80-FF`
```

Comment:
@khronokernel just noticed that there's a missing space between `* e.g. `rtcfx_exclude=00-7F` fixes`